.env

```diff
@@ -1,3 +1,12 @@
 # This is for the transit API
 REACT_APP_API_SERVER_URL=http://localhost:8001
 REACT_APP_GOOGLE_MAPS_API_KEY=AIzaSyCD3OBB8YjstMzmlR1vSfnROTWgz9sLi0E
+
```

Comment:
@meghanprestemon are these changes necessary?

Reply:
At 3:56pm Josh slacked us saying to add this to the .env in the dashboard-spa.